inc/TRestGeant4AnalysisProcess.h

```diff
@@ -80,6 +80,9 @@ class TRestGeant4AnalysisProcess : public TRestEventProcess {
     Bool_t fPerProcessSensitiveEnergy = false;
     Bool_t fPerProcessSensitiveEnergyNorm = false;
 
+    TVector3 fPrismCenter;
```

Comment:
One line description, something as:  `/// This is the center of the prism used to calculate the X observable`